### PR TITLE
lwIP: reference upstream patches for ND6 LLADDR

### PR DESCRIPTION
**Pull request by cecille**
This was fixed recently on ToT, so we should use those patches to keep everyone more in sync

### CODE REVIEW

**Review by arkq — APPROVED** [review]

File: src/lwip/patches/README.md [hunk 1]
```diff
@@ -21,11 +21,13 @@ recent API change on upstream LwIP. The previous version of this function is
...
-LLADDR option with a hw address that is not the max length. This happens for
-thread devices.
+In earlier version of lwIP, there is a bug where the RA processing fails if the
+RA includes an LLADDR option with a hw address that is not the max length. This
+happens for thread devices. Relevant patches from lwIP main that should be
+applied back to any Matter device using an older version of lwIP:
 
--   patch file: nd6_lladdr_fix.patch
+https://git.savannah.nongnu.org/cgit/lwip.git/commit/?id=7807f706f357d2eea85d9927d970991dc4b1ef90
+https://git.savannah.nongnu.org/cgit/lwip.git/commit/?id=7807f706f357d2eea85d9927d970991dc4b1ef90
```
Comment (tcarmelveilleux) [inline src/lwip/patches/README.md:30]: Both of these line are the same commit. Is that intended?

Comment (tcarmelveilleux) [inline src/lwip/patches/README.md:30]: Both of these lines are the same commit. Is that intended?

**Review by tcarmelveilleux — APPROVED** [review]

File: src/lwip/patches/README.md (hunk 1 above)
Comment (cecille) [reply src/lwip/patches/README.md]: This is what happens when you don't press ctrl-s before committing...fixed now.

Comment (cecille) [reply src/lwip/patches/README.md]: nope, fixed.

**Pull request merged**